static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -101,13 +101,17 @@ const getRules = (): Object => {
     return {
         bodySelector: '.js-article__body',
         slotSelector: ' > p',
-        minAbove: isBreakpoint({
-            max: 'tablet',
-        })
-            ? 300
-            : 700,
+        minAbove: 300,
```

Comment:
It might be worth commenting on why this is here (i.e. as a backup in case the header is missing).